### ISSUE TITLE
Getting the threshold of a custom logger

### ISSUE THREAD

**Issue by jefferys** [zatonovo/futile.logger#40]
Is it true that `flog.threshold(name='myLogger')` is supposed to return the name of the custom logger? If so it does not seem to work. Using a cloned repository I tried the following additional tests:
``` r
 flog.threshold(ERROR)
 expect_that(flog.threshold() == "ERROR", is_true())  ### WORKS
 flog.threshold(DEBUG, name='my.package')
 expect_that(flog.threshold(name='my.package') == DEBUG, is_true())  ### FAILS
```
If this is not how this is supposed to work, is there some other way get this information?

**Comment by muxspace** [comment]
Yes, it looks like you've found a bug.

**Comment by iwccsbfb** [comment]
Fixed in  PR #45
